pkg/registry/core/shoot/storage/binding.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gardener/gardener/pkg/apis/core"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apiserver/pkg/registry/rest"
+	"k8s.io/apiserver/pkg/storage"
+	"k8s.io/apiserver/pkg/util/dryrun"
+)
+
+// BindingREST implements a RESTStorage for shoots/binding.
+type BindingREST struct {
+	shootStorage *REST
+}
+
+var _ = rest.NamedCreater(&BindingREST{})
+
+// New returns and instance of Binding
+func (r *BindingREST) New() runtime.Object {
+	return &core.Binding{}
+}
+
+// Create binds a shoot to a seed
+func (r *BindingREST) Create(ctx context.Context, name string, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (out runtime.Object, err error) {
```

Comment:
Does it make sense to write a unit test for this function?

Reply:
Yes, will do.

Reply:
This is not added yet.